### PR TITLE
add ory, setup esbuild, initial migration

### PR DESCRIPTION
**Pull request by breadchris**

### CODE REVIEW

File: frontend/esbuild.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          '.ts': 'tsx',
+          '.tsx': 'tsx',
+          '.woff2': 'file',
+          '.woff': 'file',
+        },
+        plugins: [inlineImage()],
+        minify: false,
+        sourcemap: 'inline',
+        define: {
+          'global': 'window',
```
Comment (ajvpot) [inline frontend/esbuild.js:29]: duplicated defines

File: backend/pkg/actions/submitflag/submitflag.go [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+	Input            submitFlagArgs         `json:"input"`
+}
+
+type GraphQLError struct {
+	Message string `json:"message"`
+}
+
+// TODO (chthompson) oh god why
+var (
+	db *sql.DB
```
Comment (ajvpot) [inline backend/pkg/actions/submitflag/submitflag.go:29]: if you're using actions, you shouldn't need a connection to the database.

File: backend/cmd/generate.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"github.com/go-jet/jet/v2/generator/postgres"
+	_ "github.com/lib/pq"
+)
+
+func main() {
+	err := postgres.GenerateDSN(
```
Comment (ajvpot) [inline backend/cmd/generate.go:9]: I really dont like how this introduces a circular dependency. Can we avoid using codegen like this? Maybe move the source of truth to protobuf and generate the sql from that or something.

**Review by ajvpot — CHANGES_REQUESTED** [review]

File: backend/cmd/generate.go (hunk 3 above)
Comment (breadchris) [reply backend/cmd/generate.go:9]: what is the difference between this and what hasura needs to generate its metadata?

File: backend/pkg/actions/submitflag/submitflag.go (hunk 2 above)
Comment (breadchris) [reply backend/pkg/actions/submitflag/submitflag.go:29]: hasura suggests having actions call back into hasura from the action when you want data https://github.com/hasura/hasura-actions-examples/blob/master/data-validations/inventory-check/functions/nodejs-express/src/handlers/createCartItem.js#L3 which is not ideal if you want to eject from hasura in the future. SQL queries are written here to not incur that debt.

File: backend/cmd/generate.go (hunk 3 above)
Comment (ajvpot) [reply backend/cmd/generate.go:9]: fine maybe hasura is just bad

Comment (ajvpot) [reply backend/cmd/generate.go:9]: Idk how you're supposed to declarative manage your database schema, I guess I'll look into that more

**Pull request merged**